docs/tutorial/tutorial-5/android.rst

```diff
@@ -2,9 +2,234 @@
+      ...
+      [helloworld] Application created.
+
+  .. group-tab:: Windows
+
+    .. code-block:: doscon
+
+      (beeware-venv) C:\...>briefcase create android
+
+      [helloworld] Generating application template...
+      Using app template: https://github.com/beeware/briefcase-android-gradle-template.git
+      ...
+      [helloworld] Installing support package...
+      ...
+      [helloworld] Installing dependencies...
+      ...
+      [helloworld] Installing application code...
+      ...
+      [helloworld] Installing application resources...
+      ...
+      [helloworld] Application created.
+
+Once this completes, we'll now have an ``android`` directory in your project.
+This directory will contain a ``Hello World`` folder, which will contain an
+Android project with a Gradle build configuration. This project will contain
+your application code, and a support package containing the Python interpreter
+and the `rubicon-java` library.
+
+In the process, Briefcase downloads the Android SDK. You may be prompted to
+accept Google's Android SDK license.
+
+You can then use Briefcase's ``build`` command to compile this into an Android
+APK app file. It will take quite a few minutes the first time, depending on
+your Internet connection speed, and then should take less than 30 seconds every
+successive time.
+
+.. admonition:: Gradle can look stuck
+
+  You may see Gradle stuck on the "CONFIGURING" step for many, many minutes.
+  Gradle is likely busy performing downloads, and you should not interrupt it.
+  This typically only affects the first time an app is built.
+
+.. code-block:: bash
+
+  (beeware-venv) $ briefcase build android
+  [helloworld] Building Android APK...
+  Starting a Gradle Daemon
+  ...
+  BUILD SUCCESSFUL in 1m 1s
+  28 actionable tasks: 17 executed, 11 up-to-date
+  [helloworld] Built android/Hello World/app/build/outputs/apk/debug/app-debug.apk
+
+Run the app on a virtual device
+===============================
+
+We're now ready to run our application. You can use Briefcase's ``run`` command
+to run the app on an Android device, either physical or virtual. This section
+covers running the app on a virtual Android device.
+
+On Android, Briefcase's ``run`` command requires that you specify which device to run on.
+If you use ``run`` without specifying a device, Briefcase will show a list of
+physical and virtual devices (if any) and offer to create one.
+
+.. tabs::
+
+  .. group-tab:: macOS
+
+    .. code-block:: bash
+
+      (beeware-venv) $ briefcase run android
+
+      Select device:
+
+        1) Create a new Android emulator
+
+      >
+
+  .. group-tab:: Linux
+
+    .. code-block:: bash
+
+      (beeware-venv) $ briefcase run android
+
+      Select device:
+
+        1) Create a new Android emulator
+
+      >
+
+  .. group-tab:: Windows
+
+    .. code-block:: doscon
+
+      (beeware-venv) C:\...>briefcase run android
+
+      Select device:
+
+        1) Create a new Android emulator
+
+      >
+
```

Comment:
Worth adding a note here that if you have a history of doing Android development, or you have your physical device plugged in, this list might mention other devices, but "Create a new..." should always be last.

Reply:
Fixed in b663490